march_rqt_note_taker/src/march_rqt_note_taker/notes_widget.py

```diff
@@ -81,6 +82,25 @@ def _delete_selected(self):
             if indices and indices[0].isValid():
                 self._model.remove_rows(indices[0].row(), len(indices))
 
+    def _edit_selected(self):
+        selection_model = self.table_view.selectionModel()
+        if self.table_view.hasFocus() and selection_model.hasSelection():
+            indices = [index for index in selection_model.selectedIndexes() if not index.column()]
+            if indices and indices[0].isValid():
+                """ Get the full entry from dataset """
+                entry = self._model.data(indices[0], QtCore.Qt.EditRole)
+                """ Get new value for entry """
+                entry_content, ok = QInputDialog().getText(self, 'Edit Entry', 'Entry:',
+                                                           QLineEdit.Normal, entry.content)
+                if ok and entry_content:
+                    entry.edit_content(entry_content)
```

Comment:
```suggestion
                    entry.content = entry_content
```
The pythonic way is to just directly edit the parameter. Parameters that you want to be "private" should be marked by a leading _, If you need "getters/setters" for these variables, you can use @property to keep the syntax pythonic.